frontend/packages/client/src/components/Community/ProposalThresholdEditor/FormConfig.js

```diff
@@ -30,7 +30,11 @@ const Schema = (isValidFlowAddress) =>
       .when('onlyAuthorsToSubmitProposals', {
         is: true,
         then: addresValidation(yup.string(), isValidFlowAddress, true),
-        otherwise: addresValidation(yup.string(), isValidFlowAddress, false),
+        otherwise: addresValidation(
+          yup.string().required('Please enter a contract address'),
+          isValidFlowAddress,
+          false
+        ),
```

Comment:
when all fields are empty this makes the required go first, then all error messages are about fields being required
<img width="642" alt="Screen Shot 2022-08-16 at 17 03 37" src="https://user-images.githubusercontent.com/7526740/184974336-def74e2e-460b-4e99-a2b5-efec20609864.png">